### ISSUE TITLE
Logview dependency is not available on NonGNU ELPA

### ISSUE THREAD

**Issue by bbatsov** [clojure-emacs/cider#3365]
I just noticed that the ELPA CIDER build is broken because logview (https://github.com/doublep/logview) is actually only MELPA. I was under the impression this was some built-in package or whatever - I guess I misread something. Now we'll have to submit it in short order to NonGNU ELPA to fix the build there (a patch needs to be sent to the Emacs devel mailing list). 

@doublep Can you please help with this, as usually they prefer those patches to come from the package maintainers?

//cc @r0man

**Comment by bbatsov** [comment]
@doublep It's super simple - I wrote an article about the process a while ago https://emacsredux.com/blog/2021/08/11/submitting-a-package-to-nongnu-elpa/

**Comment by doublep** [comment]
Well, OK, I'll try, but note that Logview itself has some dependencies not available on NonGNU...

    paul@gonzo:~/logview$ eldev dtree
    logview 0.16.3snapshot
        datetime 0.6.1    [0.7.2 installed]
            extmap 1.1.1    [1.2.1 installed]
        extmap 1.0    [1.2.1 installed]

I guess I'll have to start with `extmap` and work my way up then?

**Comment by bbatsov** [comment]
@doublep Yeah, all the deps need to be there. Generally it's fine to submit all of them in a single PR.

**Comment by bbatsov** [comment]
@doublep Any progress here?

**Comment by bbatsov** [comment]
That's fine, I know what you mean. Not the biggest fan of emacs-devel, but I've learned to endure there. :D

**Comment by bbatsov** [comment]
Nice! It's good there are people like Stefan to help out with things like these.

**Comment by r0man** [comment]
Nice! Thanks @doublep and @bbatsov for making it happen
